app/models/katello/content_view.rb

```diff
@@ -111,6 +111,27 @@ class ContentView < Katello::Model
     scoped_search :on => :composite, :complete_value => true
     scoped_search :on => :generated_for, :complete_value => true
     scoped_search :on => :default # just for ordering
+    scoped_search :on => :name, :complete_value => true,
+                  :rename => :content_views,
```

Comment:
I considered naming this `components`, but we've moved away from using the "component" terminology recently, and I don't want to reverse that.